sig/datadog/tracing/span_operation.rbs

```diff
@@ -58,7 +58,7 @@ module Datadog
 
       def start: (?untyped? start_time) -> self
 
-      def stop: (?untyped? stop_time) -> self?
+      def stop: (?untyped? stop_time, ?Exception? error) -> self?
```

Comment:
Don't want to block you, but could you provide the type for `stop_time`?

Reply:
adressed in 791f59c9dd1972d5d39ef90d407bda431065436a